src/subscription_manager/utils.py

```diff
@@ -281,7 +281,7 @@ def get_server_versions(cp, exception_on_timeout=False):
 
     if cp:
         try:
-            supported_resources = get_supported_resources()
+            supported_resources = get_supported_resources(uep=cp)
```

Comment:
This change reduce number of TLS handshakes. Without this change: one no auth connection was used for gathering list of supported resources and then another auth/consumer connection was used fot getting status.